src/context/UserContext.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  createContext,
+  useContext,
+  useReducer,
+  ReactNode,
+  Dispatch,
+} from 'react';
+
+interface UserState {
+  name: string;
+  email: string;
+  isLoggedIn: boolean;
+  currency: string;
+  language: string;
```

Comment:
Same for language, let's add an `enum` based on the information here: https://www.figma.com/design/6KSlLFezGkD8uZXhuTsx3j/Montra---Expense-Tracker-UI-Kit-(Community)?node-id=768-3787&node-type=FRAME&t=qKzsJLWaWTjDg99b-0

Reply:
Fixed